sections/combat.tex

```diff
@@ -1,93 +1,159 @@
+\en{\hypertarget{Combat}{Combat}\index{Combat} with \textbf{Neutral Units} starts when a Hero moves to an unvisited Field with a roman numeral, signifying the \hyperlink{Difficulty}{type and number} of Neutral Units guarding that Field.}
+\pl{}
 
-Combat with \textbf{another player} can start in two ways:
+\en{Combat with \textbf{another player} can start in two ways:}
+\pl{}
 \begin{itemize}
-  \item You move into any Field containing one of their Heroes.
-  \item You move into a Town or Settlement owned by them.
+  \en{\item You move into any Field containing one of their Heroes.}
+  \pl{\item}
+  \en{\item You move into a Town or Settlement owned by them.}
+  \pl{\item}
 \end{itemize}
-Players are able to start multiple Combats during their Turn.
+\en{Players are able to start multiple Combats during their Turn.}
+\pl{}
 
-\subsection*{\hypertarget{Combatsetup}{Combat Setup}}
+\en{\subsection*{\hypertarget{Combatsetup}{Combat Setup}}}
+\pl{}
 
 % TODO info about battlefield expansion
-Combat is resolved on the 4 x 5 Combat board, which consists of two Backlines and two Frontlines on opposite ends, and a middle row.
-Follow these steps when Combat begins against \textbf{Neutral Units}:
+\en{
+  Combat is resolved on the 4 x 5 Combat board, which consists of two Backlines and two Frontlines on opposite ends, and a middle row.
+  Follow these steps when Combat begins against \textbf{Neutral Units}:
+}
+\pl{}
 
 \begin{itemize}
-  \item Choose one of the Combat Board's sides as your own.
-    Place up to 5 of your Unit cards freely onto the Back and Frontlines of that side.
-  \item Check the \hyperlink{Difficulty}{\textbf{Difficulty Table}} and draw the corresponding number of Neutral Unit cards from their Decks.
-  \item The Neutral Units are placed differently depending on the Game Mode:
-  \item In \textbf{Clash} or \textbf{Alliance} Scenarios, the enemy player sitting to your right controls the Neutral Units and decides their placement.
-    \includesvg[height=10px]{\svgs/unit_ranged.svg} Units must be placed in the Backline if possible.
-  \item In \textbf{Campaign} or \textbf{Cooperative} scenarios, Neutral Units are placed from left to right from the player's perspective.
-First, place any \includesvg[height=10px]{\svgs/unit_ranged.svg} Units in the Backline\index{Units Placement}.
-Then, place any \includesvg[height=10px]{\svgs/unit_ground.svg} or \includesvg[height=10px]{\svgs/unit_flying.svg} Units in the Frontline.
-If there's not enough room to place a Unit in its correct line, place them in the other one.
-Units must be placed in \textbf{descending} Initiative\index{Initiative} order.
-If there's a tie, place higher tier Units first.
-If there's still a tie, the players decide the order.
+  \en{
+    \item Choose one of the Combat Board's sides as your own.
+      Place up to 5 of your Unit cards freely onto the Back and Frontlines of that side.
+  }
+  \pl{\item}
```

Comment:
I'm thinking about a quick way to find untranslated strings. If we search by \pl{} we won't find all occurrences because of this case for example. is it possible and worth having an empty \language{} element always, or if not maybe replacing the whole string with a literal like TTR (to translate)? thinking about looking for untranslated strings which there will be many when a new language starts off

Reply:
Yes, good idea. A PR with this is welcome. I put those tags with `\item`, because LaTeX returns an error, if there's `\begin{itemize}` and no `\item` inside.